Apps/Admin/Tests/Functional/cypress/integration/e2e/patient-details.cy.js

```diff
@@ -84,7 +84,7 @@ function validateCovid19TreatmentAssessmentFormBackCancel() {
     cy.url().should("include", "/patient-details");
     cy.get("[data-testid=start-covid-19-treatment-assessment-button]").click();
     cy.url().should("include", "/covid-19-treatment-assessment");
-    cy.scrollTo("bottom");
+    cy.scrollTo("bottom", { ensureScrollable: false });
```

Comment:
Why is the scrolling needed at all?

Reply:
Previously blocked datasets and messaging verifications were displayed before roles had been applied. I left in the scroll in case the role implementation changed and/or if something else was put in there afterwards and/or the individual had a ton of immunizations.